cfgov/jinja2/v1/events/_macros.html

```diff
@@ -93,7 +93,7 @@
         {% endmacro %}
 
         {% macro _zip() %}
-            <span class="event-meta_zip p-postal-code">{{ zip }}</span>
+            <span class="event-meta_zip p-postal-code">{{ '%05d' | format(zip) }}</span>
```

Comment:
@jimmynotjim no,  anything less than 5 digits will will get padded with zeroes? So `232` will become `00232` and so on.